db/migrate/20210518090338_fix_srid_on_areas.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class FixSridOnAreas < ActiveRecord::Migration[6.0]
+  def up
+    %i[authority community county district instance observation].each do |table|
+      change_column table, :area, :geometry, limit: { srid: 4326, type: :multi_polygon }
+    end
+    change_column :issue, :position, :geometry, limit: { srid: 0, type: :st_point }
```

Comment:
Ist es absicht hier srid 0 zu lassen?
```suggestion
    change_column :issue, :position, :geometry, limit: { srid: 4326, type: :st_point }
```